src/main/java/com/coherentsolutions/training/auto/web/pashkovskaya/pages/WomenClothesPage.java

```diff
@@ -0,0 +1,52 @@
+package com.coherentsolutions.training.auto.web.pashkovskaya.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class WomenClothesPage extends BasePage{
+    @FindBy(xpath = "//ol[@class='product-items widget-product-grid'][1]")
```

Comment:
only one product?

Reply:
I need one of any product to test adding to the wish list or to the cart. This locator is the locator of the first available product in the list. 

As I understood, it doesn't matter which product I choose, so I chose the first one